src/wrapped32/wrappedlibc_private.h

```diff
@@ -2045,7 +2045,7 @@ GOWM(writev, lFEipi)
 // xdr_authdes_cred
 // xdr_authdes_verf
 // xdr_authunix_parms
-GO(xdr_bool, iFpp)
+//GO(xdr_bool, iFpp)
```

Comment:
what was wrong on those?

Reply:
I believe I commented this since the `XDR` structure is self-referencing, which would need a manual realignment.
The simplified declaration of the wrapperhelper is:
```
Declaration: xdr_bool -> ...: int32_t((* struct __rpc_xdr <with 6 members, with self-recursion> { <snip> }), (* int32_t))
```
i.e. `int32_t xddr_bool(struct __rpc_xdr*, int32_t*)` with `struct __rpc_xdr` a self-referencing structure.

Reply:
Reverted.